From f41ac3f8b68e0be9bb8426878fe7da1faca34446 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 16 Apr 2020 21:04:22 +0300
Subject: [PATCH] [SW-11770] hl: rename the bootloader fit image

The FIT image will be mostly used for loading U-Boot through PCI to the
on-chip SRAM. Sometimes it will be used to program the BTL and Preboot
to SPI flash (rarely).
To make the file name describe better the content, rename the FIT image
from
${HL_DEV}-fit-spi.itb
to
${HL_DEV}-boot-fit.itb

As a small benefit, the bash complition will be faster.

Change-Id: I189a9f3ef5d0a67780e166e96c4af5abbb4dcedb
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/post-image-fit.sh b/board/habanalabs/common/post-image-fit.sh
index b8940fd..38359cb 100755
--- a/board/habanalabs/common/post-image-fit.sh
+++ b/board/habanalabs/common/post-image-fit.sh
@@ -26,7 +26,7 @@
 	sed ${ITS_SED_PARAMS} ${UBOOT_ITS_TEMPLATE} > ${UBOOT_ITS_FILE}
 	${HOST_DIR}/usr/bin/mkimage -D "-W no-unit_address_vs_reg" -f ${UBOOT_ITS_FILE} ${UBOOT_ITB_FILE}
 	# LKD like this to be 4B aligned in size
-	dd status=none conv=sync bs=4 if=${UBOOT_ITB_FILE} > ${HL_DEV}-fit-spi.itb
+	dd status=none conv=sync bs=4 if=${UBOOT_ITB_FILE} > ${HL_DEV}-boot-fit.itb
 
 	sed ${ITS_SED_PARAMS} ${LINUX_ITS_TEMPLATE} > ${LINUX_ITS_FILE}
 	cp ${BASE_DIR}/build/linux-custom/arch/arm64/boot/Image .
diff --git a/scripts/make/common/Makefile b/scripts/make/common/Makefile
index 624b6e7..4204ee0 100644
--- a/scripts/make/common/Makefile
+++ b/scripts/make/common/Makefile
@@ -172,7 +172,7 @@
 .PHONY: hl-install-host hl-install-lab
 hl-install-host:
 	sshpass -p123456 $(SCP) $(IMAGES_DIR)/$(HL_DEV)-fit.itb		root@$(HL_INSTALL_HOST_IP):/lib/firmware/habanalabs/$(HL_DEV)/
-	sshpass -p123456 $(SCP) $(IMAGES_DIR)/$(HL_DEV)-fit-spi.itb	root@$(HL_INSTALL_HOST_IP):/lib/firmware/habanalabs/$(HL_DEV)/
+	sshpass -p123456 $(SCP) $(IMAGES_DIR)/$(HL_DEV)-boot-fit.itb	root@$(HL_INSTALL_HOST_IP):/lib/firmware/habanalabs/$(HL_DEV)/
 	sshpass -p123456 $(SCP) $(IMAGES_DIR)/$(HL_DEV)-spi-flash.bin	root@$(HL_INSTALL_HOST_IP):/lib/firmware/habanalabs/$(HL_DEV)/
 hl-install-lab:
 	-sshpass -p123456 $(SCP) $(IMAGES_DIR)/$(HL_DEV)-spi-flash.bin	root@$(HL_INSTALL_LAB_IP):/lib/firmware/habanalabs/$(HL_DEV)/
